spec/feature/user_can_create_new_account_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
```

Comment:
please see my comment above. If you modify your `.rspec` file, you no longer have to require rails_helper here.